src/Redmine/Api/Wiki.php

```diff
@@ -71,6 +71,16 @@ public function create($project, $page, array $params = [])
 
         $xml = new \SimpleXMLElement('<?xml version="1.0"?><wiki_page></wiki_page>');
         foreach ($params as $k => $v) {
+            if ('uploads' === $k && is_array($v)) {
+                 $uploadsItem = $xml->addChild('uploads', '');
+                 $uploadsItem->addAttribute('type', 'array');
+                 foreach ($v as $upload) {
+                     $upload_item = $uploadsItem->addChild('upload', '');
+                     foreach ($upload as $upload_k => $upload_v) {
+                         $upload_item->addChild($upload_k, $upload_v);
+                     }
+                 }
+             } else
             $xml->addChild($k, $v);
```

Comment:
the else case is missing curly brackets `{...}`.
Applying php-cs-fixer should fix the issue